README.md

```diff
@@ -1,24 +1,38 @@
 # Licensee
-_A Ruby Gem to detect under what license a project is distributed._
+
+*A Ruby Gem to detect under what license a project is distributed.*
 
 [![Build Status](https://travis-ci.org/benbalter/licensee.svg?branch=master)](https://travis-ci.org/benbalter/licensee) [![Gem Version](https://badge.fury.io/rb/licensee.svg)](http://badge.fury.io/rb/licensee)
 
 ## The problem
-- You've got an open source project. How do you know what you can and can't do with the software?
-- You've got a bunch of open source projects, how do you know what their licenses are?
-- You've got a project with a license file, but which license is it? Has it been modified?
+
+* You've got an open source project. How do you know what you can and can't do with the software?
+* You've got a bunch of open source projects, how do you know what their licenses are?
```

Comment:
The comma after "projects" introduces a run-on sentence. I'd suggest a period, like the line above, or a semicolon